registry/server/templates/services/templatesRepository.ts

```diff
@@ -227,6 +261,19 @@ export class TemplatesRepository {
         return { type: 'ok' };
     }
 
+    private async mustReadTemplate(templateName: string): Promise<VersionedRecord<Template>> {
```

Comment:
usual syntax for this in ORMs afaik is `get` || `findOrFail`, tbh it's slightly harder to read `mustReadTemplate`